### ISSUE TITLE
Footprint Metadata Support

### ISSUE THREAD

**Issue by davidperrenoud** [fritzing/fritzing-app#173]
_From [brendan....@gmail.com](https://code.google.com/u/110718573451811856532/) on October 13, 2008 10:42:02_

What steps will reproduce the problem? 1. Try to find a footprint file when you don't know the name of the package
or file name. What is the expected output? What do you see instead? The footprint files should have some sort of metadata in them so that parts
creators can easily find parts based on various properties as well as text
search.  
Suggested solution is to add fritzing:tag namespace tags to footprint
SVGs as well as adding intermediate UI based choices/filter in the parts
editor.  Using something like "provides/properties" would work as well.

_Original issue: http://code.google.com/p/fritzing/issues/detail?id=173_

**Comment by failiz** [comment]
I agree that this is needed in the future part editor. Not sure if metadata is the best way. Most CAD programs (f ex. KiCad) use libraries, keyword and number of pins filters.
